src/Components/ArtsyShippingEstimate.tsx

```diff
@@ -39,7 +43,29 @@ interface ArtsyShippingEstimateProps {
   artwork: ArtsyShippingEstimate_artwork$key
 }
 
-export const ArtsyShippingEstimate = ({
+export const ArtsyShippingEstimate = (props: ArtsyShippingEstimateProps) => {
+  // TODO: Arta Estimate widget experiment!
+  // This code determins if Arta widget code should be loaded
+  // Either remove or properly implement when experiment is
```

Comment:
#minor. I see this comment was copied and pasted here, but it seems the last sentence is incomplete.